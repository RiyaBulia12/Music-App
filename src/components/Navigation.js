import { Link } from 'react-router-dom';

const Navigation = () => (
  <>
    <div className="flex sticky top-0 p-8 bg-slate-700 text-white justify-between items-center lg:py-8 lg:px-24">
      <Link to="/">
        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth="1.5" stroke="currentColor" className="w-6 h-6">
          <path strokeLinecap="round" strokeLinejoin="round" d="M15.75 19.5L8.25 12l7.5-7.5" />
        </svg>
      </Link>

      <div className="flex items-center">
        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor" className="w-6 h-6 mr-2">
          <path strokeLinecap="round" strokeLinejoin="round" d="M9 9l10.5-3m0 6.553v3.75a2.25 2.25 0 01-1.632 2.163l-1.32.377a1.803 1.803 0 11-.99-3.467l2.31-.66a2.25 2.25 0 001.632-2.163zm0 0V2.25L9 5.25v10.303m0 0v3.75a2.25 2.25 0 01-1.632 2.163l-1.32.377a1.803 1.803 0 01-.99-3.467l2.31-.66A2.25 2.25 0 009 15.553z" />
        </svg>
        <h1 className="text-2xl">Musicana</h1>
      </div>
      <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor" className="w-6 h-6">
        <path strokeLinecap="round" strokeLinejoin="round" d="M21 21l-5.197-5.197m0 0A7.5 7.5 0 105.196 5.196a7.5 7.5 0 0010.607 10.607z" />
      </svg>
    </div>
  </>
);

export default Navigation;
