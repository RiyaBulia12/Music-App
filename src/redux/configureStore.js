import { combineReducers, configureStore } from '@reduxjs/toolkit';
import { playlistReducers } from './reducers/playlistReducers';

const reducers = combineReducers({ playlist: playlistReducers });
const store = configureStore({ reducer: reducers });

export default store;
