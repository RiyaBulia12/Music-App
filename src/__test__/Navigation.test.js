import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import Navigation from '../components/Navigation';

describe('Navigation Test', () => {
  const initialState = { albumName: 'ABC', counts: 230 };
  const mockStore = configureStore();
  let store;

  const MockNav = () => {
    store = mockStore(initialState);
      <Provider store={store}>
        <Navigation />
      </Provider>;
  };

  test('renders navigation correctly', () => {
    expect(MockNav).toMatchSnapshot();
  });
});
