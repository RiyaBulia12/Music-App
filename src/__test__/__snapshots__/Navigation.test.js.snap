// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation Test renders learn react link 1`] = `[Function]`;

exports[`Navigation Test renders navigation correctly 1`] = `[Function]`;
